client/src/components/profile-picture/profile-picture.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import './profile-picture.scss';
+import EmptyProfilePicture from './assets/empty_profile_picture.svg';
+
+interface ProfilePictureProps {
+    profilePictureSrc: string | null,
+}
+
+const ProfilePicture: FC<ProfilePictureProps> = (props) => {
+
+    return (
+        <div
```

Comment:
Perhaps you could render an img-tag enclosed by a div-tag, in order to follow html semantics and increase code readability?